editor/editor_inspector.cpp

```diff
@@ -2243,6 +2243,14 @@ void EditorInspector::_edit_set(const String &p_name, const Variant &p_value, bo
 		undo_redo->add_do_property(object, p_name, p_value);
 		undo_redo->add_undo_property(object, p_name, object->get(p_name));
 
+		PropertyInfo prop_info;
+		if (ClassDB::get_property_info(object->get_class_name(), p_name, &prop_info)) {
+			for (int linked_prop_index = 0; linked_prop_index < prop_info.linked_properties.size(); ++linked_prop_index) {
```

Comment:
You can use range loop instead
`for (const String &linked_prop : prop_info.linked_properties) {`

Reply:
Thanks, done!